cli/src/main/scala/com/ibm/sparktc/sparkbench/workload/ml/MLWorkloadResults.scala

```diff
@@ -0,0 +1,9 @@
+package com.ibm.sparktc.sparkbench.workload.ml
+
+case class MLWorkloadResults(
```

Comment:
I'm pretty sure this file can get deleted, this code isn't getting touched anywhere. That may be taken care of in my latest changes in my branch though.